### ISSUE TITLE
Shared_dirs doesn't seem right

### ISSUE THREAD

**Issue by superdav42** [jalogut/magento2-deployer-plus#14]
In recipe/magento_2_1.php shared_dirs are set as:
```
set('shared_dirs', [
    '{{magento_dir}}/var',
    '{{magento_dir}}/pub/media',
    '{{magento_dir}}/pub/static/_cache',
]);
```
Seems like it the media folder should be the only shared dir. 
Commands like 
```
bin/magento deploy:mode:set production
Enabling maintenance mode
Config "dev/debug/debug_logging = 0" has been saved.
Disabling maintenance mode
The directory "/srv/releases/20180628170504/pub/static/_cache" cannot be deleted Warning!rmdir(/srv/releases/20180628170504/pub/static/_cache): Not a directory
```
fail when pub/static/_cache is a symlink and I see no advantage to making it a shared dir.

Also I would think /var shouldn't be a shared dir as any cache clearing or setting in one release would affect all the others causing all possible problems with version mismatches.

**Comment by jalogut** [comment]
Hi @superdav42 yes you are right. I am aware of that but I did not fix it yet. Var should not be shared but some folders inside need it. I am gonna add a fix next week for that.

Regarding the `_cache` why are you executing `deploy:mode:set production`? That should not be needed. The reason why this folder is shared is here:

- https://github.com/jalogut/magento2-deployer-plus/pull/12

**Comment by robin31** [comment]
We have the same issue with CSS compiling because of this shared dir:

```
➤ Executing task files:compile
[stage_master] > cd /var/vhost/XXXXX/releases/20180704110031 && (/usr/bin/php ./bin/magento setup:di:compile)
[stage_master] < The directory "/var/vhost/XXXXX/releases/20180704110031/var/cache" cannot be deleted Warning!rmdir(/var/vhost/XXXX/releases/20180704110031/var/cache): Not a directory
```

**Comment by jalogut** [comment]
@superdav42 fix has been added in version `2.0`, please let me know if you still have issues

**Comment by jalogut** [comment]
I managed to reproduce this issue and add a PR that fixes it. It should be available soon.

- https://github.com/jalogut/magento2-deployer-plus/pull/18

**Comment by jalogut** [comment]
Fixed in version `2.1`
